app-tracking-protection/vpn-internal/src/main/java/com/duckduckgo/vpn/internal/feature/rules/ExceptionRulesDebugReceiver.kt

```diff
@@ -48,7 +49,7 @@ class ExceptionRulesDebugReceiver(
 
     init {
         kotlin.runCatching { context.unregisterReceiver(this) }
-        context.registerReceiver(this, IntentFilter(intentAction))
+        ContextCompat.registerReceiver(context, this, IntentFilter(intentAction), ContextCompat.RECEIVER_NOT_EXPORTED)
```

Comment:
nit: maybe we want to add all this calls to as an extension function in `ContextExtensions.kt`, ie
```kotlin
fun Context.registerNotExportedReceiver(receiver: BroadcastReceiver, intentFilter: IntentFilter) {
  ContextCompat.registerReceiver(this, receiver, intentFilter, ContextCompat.RECEIVER_NOT_EXPORTED)
}

fun Context.registerExportedReceiver(receiver: BroadcastReceiver, intentFilter: IntentFilter) {
  ContextCompat.registerReceiver(this, receiver, intentFilter, ContextCompat.RECEIVER_EXPORTED)
}
```